libavfilter/f_ebur128.c

```diff
@@ -748,6 +750,15 @@ static int filter_frame(AVFilterLink *inlink, AVFrame *insamples)
                     }
                 }
             }
+            else if (!ebur128->gate_measurement) {
+                ebur128->i400.sum_kept_powers += power_400;
+                ebur128->i400.nb_kept_powers++;
+                ebur128->integrated_loudness = LOUDNESS(ebur128->i400.sum_kept_powers / ebur128->i400.nb_kept_powers);
```

Comment:
Is there a reason in the ungated logic we're using the power of the 400 bin, where gated we're using the energy?

Reply:
"
With the pre-filter and the RLB filtering applied, the mean-square energy in the measurement
interval T is then measured as:
"
The above description is from the page:4 of [https://www.itu.int/dms_pubrec/itu-r/rec/bs/R-REC-BS.1770-1-200709-S!!PDF-E.pdf](url)


I think we can assume T is 100ms (When calculating the loudness_400, it is 400ms long with 75% overlap so computation is needed each 100ms),   which is not conflict with the standard.

But I am not sure if can keep the `75% overlap` logic there.

Reply:
That's probably fine I think?

Reply:
From my mind, it is okay, because integrated loudness is just the average value of samples and overlap won't affect the result.

Reply:
Both power and energy are the same thing.